.dockerignore

```diff
@@ -0,0 +1,2 @@
+*
+!docker/
```

Comment:
I switched from `docker build -f docker/Dockerfile docker` to `docker build .` because it requires less special treatment. When using Jenkins model definition plug-in it didn't support specifying a custom directory for example, which made docker pass to the docker daemon the full repository (which is a lot of copying). I think is better for the project to be more self-sufficient. Any casual user that doesn't know about the details on how docker works would be much safer if the obvious way to build an image for the project "just works" without needing to use special rules to build it.